api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -48,6 +54,40 @@ message RedisProxy {
     bool enable_hashtagging = 2;
   }
 
-  // Network settings for the connection pool to the upstream cluster.
+  // Network settings for the connection pool to the upstream clusters.
   ConnPoolSettings settings = 3 [(validate.rules).message.required = true];
+
+  message PrefixRoute {
+    // String prefix that must match the beginning of the keys. Envoy will always favor the longest
+    // matches.
+    string prefix = 1 [(validate.rules).string.min_bytes = 1];
+
+    // Indicates if the prefix needs to be removed from the key when forwarded.
+    bool preserve_prefix = 2;
```

Comment:
Should the default be true here? Seems like most people would want that? If that's the case negate to `remove_prefix` ?